oidc-samples/org.wso2.identity.sdk.oidc/org.wso2.identity.sdk.oidc/obj/org.wso2.identity.sdk.oidc.csproj.nuget.dgspec.json

```diff
@@ -0,0 +1,66 @@
+{
+  "format": 1,
+  "restore": {
+    "C:\\Users\\wso2\\source\\repos\\Project - IAM\\org.wso2.identity.sdk.oidc\\org.wso2.identity.sdk.oidc\\org.wso2.identity.sdk.oidc.csproj": {}
```

Comment:
We can't have absolute paths. Shall we check a way to have relative paths.

Reply:
Fixed.